common/src/test/scala/org/neo4j/spark/service/Neo4jQueryServiceTest.scala

```diff
@@ -38,14 +42,17 @@ import scala.collection.immutable.HashMap
 
 class Neo4jQueryServiceTest {
 
+  private val neo4j: Neo4j =
+    new Neo4j(new Neo4jVersion(4, 4, 0), Neo4jEdition.ENTERPRISE, Neo4jDeploymentType.SELF_MANAGED)
```

Comment:
Do we also need to add some tests for later versions that end up in `CYPHER 5` prefix?

Reply:
added in 069e9bf99d65e506c03240d67ce6bbea6532bdf7